content/en/synthetics/cicd_integrations/configuration.md

```diff
@@ -0,0 +1,387 @@
+
+`ORIGIN`
+: Test's origin<br>
+**Example**: `https://www.example.org:81`
+
+`PARAMS`
+: Test's query parameters<br>
+**Example**: `?abc=123`
+
+`PATHNAME`
+: Test's URl path<br>
+**Example**: `/path/to/something`
+
+`PORT`
+: Test's host port<br>
+**Example**: `81`
+
+`PROTOCOL`
+: Test's protocol<br>
+**Example**: `https:`
+
+`SUBDOMAIN`
+: Test's sub domain<br>
+**Example**: `www`
+
+Whether you use Synthetic tests to control your CI/CD deployments in production or staging, you can run Synthetic tests against a generated staging URL instead of in production by setting local environment variables in your test's starting URL. 
+
+To trigger an existing Synthetics test on a staging endpoint instead of in production, set the `$SUBDOMAIN` environment variable to `staging-example` and the `$PORT` environment variable to a port used for staging. Your Synthetic tests run against the generated staging URL instead of running in production. 
+
+For example, you can write `https://www.example.org:81/path/to/something?abc=123#target` as:
+
+* `{{PROTOCOL}}//{{SUBDOMAIN}}.{{DOMAIN}}:{{PORT}}{{PATHNAME}}{{PARAMS}}{{HASH}}`
+* `{{PROTOCOL}}//{{HOST}}{{PATHNAME}}{{PARAMS}}{{HASH}}`
+* `{{URL}}`
+
+**Note:** If you have environment variables with names corresponding to one of the reserved variables above, your environment variables are ignored and replaced with the corresponding component parsed from your test's `startUrl`. 
+
+### Run tests
+
+You can decide to have the CLI auto-discover all your `**/*.synthetics.json` Synthetic tests (or all the tests associated to the path specified in your [global configuration file](#setup-a-client)) or to specify the tests you want to run using the `-p,--public-id` flag.
+
+Run tests by executing the CLI:
+
+{{< tabs >}}
+{{% tab "Yarn" %}}
+
+```bash
+yarn datadog-ci synthetics run-tests
+```
+
+**Note**: If you are launching your tests with a custom global configuration file, append your command with `--config <PATH_TO_GLOBAL_CONFIG_FILE`.
+
+{{% /tab %}}
+{{% tab "NPM" %}}
+
+Add the following to your `package.json`:
+
+```json
+{
+  "scripts": {
+    "datadog-ci-synthetics": "datadog-ci synthetics run-tests"
+  }
+}
+```
+
+Then, run:
+
+```bash
+npm run datadog-ci-synthetics
+```
+
+**Note**: If you are launching your tests with a custom global configuration file, append the command associated to your `datadog-ci-synthetics` script with `--config <PATH_TO_GLOBAL_CONFIG_FILE`.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Use the testing tunnel
+
+The [@datadog/datadog-ci][1] NPM package also comes with a tunnel functionality, allowing you to trigger Synthetic tests on your internal applications. 
+
+The testing tunnel creates an end-to-end encrypted HTTP proxy between your infrastructure and Datadog that allows all test requests sent through the CLI to be automatically routed through the `datadog-ci` client. 
+
+This allows Datadog to run tests on your internal applications. For more information, see [Testing Tunnel][4].
```

Comment:
We're repeating ourselves here with the first sentence, any alternatives we could use? Or just shorten it to `For more information, see [Testing Tunnel][4].`?